RegExp.make.js

```diff
@@ -160,12 +154,9 @@ RegExp.make = (function () {
     /** @type {!Array.<number>} */
     const ranges = this.ranges;
     /** @type {number} */
-    const n = ranges.length;
-    var pastLastRight = 0;
+    let pastLastRight = 0;
```

Comment:
Is `let` implemented in Firefox or should I change `check.sh` to run with `--language_out=ecmascript5` so we run the tests under transpilation?